runtime/pangolin/src/migration.rs

```diff
@@ -49,6 +53,15 @@ fn migrate() -> frame_support::weights::Weight {
 	migration::clear_storage_prefix(b"TechnicalMembership", b"Members", &[], None, None);
 	migration::clear_storage_prefix(b"TechnicalMembership", b"Prime", &[], None, None);
 
+	const REVERT_BYTECODE: [u8; 5] = [0x60, 0x00, 0x60, 0x00, 0xFD];
+	// CONVICTION_VOTING_ADDRESS equals to the addr(1538) in the pallet-evm runtime.
+	const CONVICTION_VOTING_ADDRESS: &str = "0x0000000000000000000000000000000000000602";
+	EVM::create_account(
```

Comment:
Why do we need to create an account for this?

Reply:
In addition to inserting to the `AccountCode`, the `create_account` function also follows the substrate account management rule to increase the corresponding account references, just like normal smart contract accounts.

Reply:
Do we need to create an account for any new precompile added?

Reply:
Yeah.